string id
int64 job_id
int64 sign_id
int64 time_start    # time in seconds since Unix epoch
int64 time_end      # time in seconds since Unix epoch
float64 x_coordinate
float64 y_coordinate